receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -377,7 +413,10 @@ public void testListPrepaidCardReceipt_clickTransactionDisplaysDetails() {
         onView(withId(R.id.transaction_title)).check(matches(withText(R.string.prepaid_card_sale)));
         onView(withId(R.id.transaction_amount)).check(matches(withText("- 10.00")));
         onView(withId(R.id.transaction_currency)).check(matches(withText("USD")));
-        onView(withId(R.id.transaction_date)).check(matches(withText("June 6, 2019")));
+        onView(withId(R.id.transaction_date)).check(
+                matches(withText(formatDateTime(mActivityTestRule.getActivity().getApplicationContext(),
+                        DateUtils.fromDateTimeString("2019-06-06T22:49:17").getTime(),
```

Comment:
It will be look good if we place the expected value(like June 6, 2019) instead of date & time.  I think we can achieve by this date format operation. Could you please give some opinion  @skoong .

Reply:
Yes, I agree that this is hard to read.  However we need to convert the date and time into the correct expected value which is in readable format (June 6, 2019) so not sure if thats doable here.
@vinotb-altmtrk I think you need to consider the timezone as well.

Reply:
@skoong Ok shyang. I will check whether if we can make readable format is possible.